config/ftbquests/quests/chapters/computercraft_tweaked.snbt

```diff
@@ -398,13 +409,20 @@
 			icon: "computercraft:pocket_computer_normal"
 			x: -4.5d
 			y: 4.5d
+			subtitle: "Wait there is even more?"
+			description: ["ComputerCraft itself adds a few unique peripherals.  Some of them aren't as useful as others, but all of them are worth attention ^^"]
```

Comment:
```suggestion
			description: ["ComputerCraft itself adds a few unique peripherals. Some of them aren't as useful as others, but all of them are worth attention ^^"]
```

Reply:
Ups. I will need to check entire quest line for double spaces. Forgot that my keyboard likes to do them recently hah